tests/test_util.py

```diff
@@ -11,29 +11,18 @@
 
 
 class FakeGH:
-    def __init__(self, *, getiter=None, getitem=None, post=None, patch=None):
+    def __init__(self, *, getitem=None, post=None, patch=None):
         self._getitem_return = getitem
-        self._getiter_return = getiter
         self._post_return = post
         self._patch_return = patch
         self.getitem_url = None
-        self.getiter_url = None
         self.patch_url = self.patch_data = None
         self.post_url = self.post_data = None
 
     async def getitem(self, url):
         self.getitem_url = url
         to_return = self._getitem_return[self.getitem_url]
-        if isinstance(to_return, Exception):
-            raise to_return
-        else:
-            return to_return
-
-    async def getiter(self, url):
-        self.getiter_url = url
-        to_iterate = self._getiter_return[url]
-        for item in to_iterate:
-            yield item
+        return to_return
```

Comment:
GH won't allow the suggestion, but we can combine these now:

```python
return self._getitem_return[self.getitem_url]
```

Reply:
Thanks, updated.